serviceerror/already_exists.go

```diff
@@ -20,6 +21,13 @@ func NewAlreadyExist(message string) error {
 	}
 }
 
+// NewAlreadyExistf returns new AlreadyExists error with formatted message.
+func NewAlreadyExistf(format string, args ...interface{}) error {
```

Comment:
```suggestion
// NewAlreadyExistsf returns new AlreadyExists error with formatted message.
func NewAlreadyExistsf(format string, args ...interface{}) error {
```
and ideally the non-f one should be renamed too